index.css

```
@@ -1,79 +1,119 @@
-
 * {
     padding: 0;
     margin: 0;
     box-sizing: border-box;
 }
-#firstSection {
-    margin: 0 auto;
+
+body {
+    font-size: 15px;
+    font-family: Lato;
 }
 
+/* Header styles */
+
 .pageHeader {
+    background:#0B173B ;
+}
+
+.container {
+    margin: 0 auto;
+    max-width: 1400px;
+}
+
+#firstContainer {
     display: flex;
-    background:#000080;
+    padding: 11px 50px;
+}
+
+#logo {
+    flex-basis: 20%;
+    padding: 8px;
 }
 
 #options {
-    padding: 37px 24px 0px 0px ;
-    flex-basis: 75%;
+    padding: 2px 11px 1px 0px ;
+    flex-basis: 80%;
 }
 
 .optionsList {
     display: flex;
+    color: #FAFAFA;
     justify-content: end;
     list-style: none;
 }
 
+#idiomMenu {
```

```suggestion
#languageMenu {
```
Usa `language` idiom no es correcto.